eZ/Publish/SPI/Comparison/ComparisonValue.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Comparison;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+abstract class ComparisonValue extends ValueObject
+{
+    /** @var mixed */
```

Comment:
Not sure if this will answer your question, but this is not FT-specific, is a base for FT-specific classes like that one:
`\eZ\Publish\SPI\Comparison\Value\StringComparisonValue`, but maybe we should drop `$value` from it then.

Reply:
> Not sure if this will answer your question, but this is not FT-specific, is a base for FT-specific classes like that one:
> `\eZ\Publish\SPI\Comparison\Value\StringComparisonValue`, but maybe we should drop `$value` from it then.

AFAICS `$value` is handled by specific strategy/engine, so yes, not needed here, belongs to a specific instance of ComparisonValue.